external/netty/README.md

```diff
@@ -0,0 +1,31 @@
+# External Netty Tests
+
+Netty tests are part of the external third-party application tests that help verify that the Adoptium binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/Issue #172 lists the applications that we have initially targeted to best exercise the Adoptium binaries. For each application, we choose to run a selection of their functional tests.
```

Comment:
As AdoptOpenJDK has moved to Eclipse as Adoptium please update references to AdoptOpenJDK to be Adoptium. Thanks!

Reply:
I have changed all the references of AdoptOpenJDK to Adoptium. If there is still any reference that I missed my eye, please let me know.